### ISSUE TITLE
RWD issue: lt-${breakpoint} should be `${breakpointValue - 0.1px}` 

### ISSUE THREAD

**Issue by achinchen** [unocss/unocss#707]
# Description

## TL;DR
- breakpoint: md
- breakpointValue: 768px

`lt-${breakpoint}` should be `${breakpointValue - 0.1px}` instead using ${breakpointValue} directly.

After I read the [source code](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/variants/breakpoints.ts#L37-L49), I think `at-${breakpoint}` do have the same issue. 

I could open the PR to fix it if needed.

## Situtation
I found the issue causes the user who uses screen which right `at` breakpoint will see nothing when I tried to use `lt-${breakpoint}:none` and `${breakpoint}:none` to control the different components like below sample code.

components
```jsx
const Desktop = () => <div display="flex lt-md:none">desktop</div>
const Mobile = () => <div display="flex md:none">mobile</div>
```` 
 output
```css
[display~="flex"]{display:flex;}

/* I expect to get `767.9px` instead of `768px` */
@media (max-width: 768px){
   [display~="lt-md\:none"]{display:none;}
}

@media (min-width: 768px){
   [display~="md\:none"]{display:none;}
}
```

# REF
[windicss-rwd-custom-range](https://windicss.org/features/responsive-design.html#custom-range)

**Comment by antfu** [comment]
PR welcome :)
